package.json

```
@@ -13,22 +13,22 @@
     "./use-at-your-own-risk": "./lib/unsupported-api.js"
   },
   "scripts": {
+    "build:docs:update-links": "node tools/fetch-docs-links.js",
+    "build:release": "node Makefile.js generateRelease",
+    "build:release:alpha": "node Makefile.js generatePrerelease -- alpha",
+    "build:release:beta": "node Makefile.js generatePrerelease -- beta",
+    "build:release:publish": "node Makefile.js publishRelease",
+    "build:release:rc": "node Makefile.js generatePrerelease -- rc",
```

It's a bit confusing that "publish" comes before "rc". The standard (which I wrote) says alphabetical order. May want to update the standard to make room for nuances like this. 

```suggestion
    "build:release:rc": "node Makefile.js generatePrerelease -- rc",
    "build:release:publish": "node Makefile.js publishRelease",
```

---

I understand that there are one-off cases where the ordering can be unusual, but personally, I don't think we should have exceptions to alphabetical ordering. We should just enable [prefer-alphabetical-scripts](https://npmpackagejsonlint.org/docs/rules/scripts/prefer-alphabetical-scripts) from [npm-package-json-lint](https://github.com/tclindner/npm-package-json-lint) and call it a day so there's no need to debate the ordering.

---

Makes sense. 

automation > bikeshedding

---

Running [`sort-package-json`](https://github.com/keithamus/sort-package-json)'s [`--check` flag](https://github.com/keithamus/sort-package-json#--check-flag) in CI can also help here I think